zuul.d/jobs.yaml

```diff
@@ -27,15 +27,58 @@
+
+- job:
+    name: heat-operator-tempest-multinode-ceph
+    parent: podified-multinode-hci-deployment-crc-1comp-backends
+    dependencies: ["openstack-k8s-operators-content-provider"]
+    vars:
+      cifmw_extras:
+        - "@scenarios/centos-9/multinode-ci.yml"
+        - "@scenarios/centos-9/hci_ceph_backends.yml"
+        - "@{{ ansible_user_dir }}/{{ zuul.projects['github.com/openstack-k8s-operators/heat-operator'].src_dir }}/ci/tempest-multinode-ceph/ci_fw_vars.yaml"
+      cifmw_run_test_role: test_operator
+      cifmw_test_operator_concurrency: 4
+      cifmw_test_operator_timeout: 7200
+      cifmw_test_operator_tempest_network_attachments:
+        - ctlplane
+      # This value is used to populate the `tempestconfRun` parameter of the Tempest CR: https://openstack-k8s-operators.github.io/test-operator/crds.html#tempest-custom-resource
+      # https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/test_operator/defaults/main.yml
+      # NOTE(gibi): identity.v3_endpoint_type override is a WA to force the
+      # publicURL as otherwise tempest gets configured with adminURL and that
+      # causes test instability.
+      cifmw_tempest_tempestconf_config:
+        overrides: |
+            compute-feature-enabled.vnc_console true
+            validation.run_validation true
+            identity.v3_endpoint_type public
+            identity.v2_admin_endpoint_type public
+            volume.volume_type_multiattach multiattach
+            volume.storage_protocol ceph
+            service_available.swift false
+            service_available.cinder true
+            share.run_share_group_tests false
+            share.capability_storage_protocol cephfs
+            share.suppress_errors_in_cleanup true
+      cifmw_test_operator_tempest_include_list: |
+        heat_tempest_plugin.*
+      cifmw_test_operator_tempest_exclude_list: |
+        heat_tempest_plugin.tests.functional.test_os_wait_condition.OSWaitCondition.test_create_stack_with_multi_signal_waitcondition
+        heat_tempest_plugin.tests.scenario.test_server_cfn_init.CfnInitIntegrationTest.test_server_cfn_init
+        heat_tempest_plugin.tests.scenario.test_aodh_alarm.AodhAlarmTest.test_alarm
+        heat_tempest_plugin.tests.scenario.test_server_signal.ServerSignalIntegrationTest.test_server_signal_userdata_format_raw
+        heat_tempest_plugin.tests.scenario.test_server_signal.ServerSignalIntegrationTest.test_server_signal_userdata_format_software_config
+        heat_tempest_plugin.tests.scenario.test_server_software_config.SoftwareConfigIntegrationTest.test_server_software_config
+        heat_tempest_plugin.tests.scenario.test_volumes.VolumeBackupRestoreIntegrationTest.test_cinder_volume_create_backup_restore
```

Comment:
I thought I would see less failures in the ceph job, but it is pretty similar
without ceph: https://logserver.rdoproject.org/31/431/4db24fa5f40996c29ad51f7b4b1b82a66a243a77/github-check/heat-operator-tempest-multinode/c67b143/controller/ci-framework-data/tests/test_operator/tempest-tests/stestr_results.html

ceph: https://logserver.rdoproject.org/31/431/4db24fa5f40996c29ad51f7b4b1b82a66a243a77/github-check/heat-operator-tempest-multinode-ceph/663910e/controller/ci-framework-data/tests/test_operator/tempest-tests/stestr_results.html

I wonder if it still worth enabling the ceph job

Reply:
@bshephar @rabi do you think it is worth enabling the ceph job?
I'm thinking about closing this PR

Reply:
Some of the errors are because the VM can't resolve the endpoint to signal back to Heat:
```
curl: (6) Could not resolve host: heat-api-public-openstack.apps-crc.testing
curl: (6) Could not resolve host: heat-api-public-openstack.apps-crc.testing
curl: (6) Could not resolve host: heat-api-public-openstack.apps-crc.testing
curl: (6) Could not resolve host: heat-api-public-openstack.apps-crc.testing
}}}
```

I think we're skipping these tests downstream for the same reason.

Reply:
Problem is that we don't have a DNS server that the VM can reach and resolve the hostname for. We can't use the IP because of the TLS requirements.